apps/src/aichat/constants.ts

```diff
@@ -0,0 +1,12 @@
+export const Role = {
```

Comment:
It might be worth turning these into [enums](https://www.typescriptlang.org/docs/handbook/enums.html). You can still retain the string mapping by adding values, like
```
enum Role {
  ASSISTANT = 'assistant';
  USER = 'user';
...
```
but then you can refer to this enum as a type in other interfaces (`role: Role`)